rs/registry/canister/src/registry_lifecycle.rs

```diff
@@ -169,22 +169,6 @@ mod test {
         canister_post_upgrade(&mut registry, registry_storage);
     }
 
-    #[test]
-    #[should_panic(expected = "No routing table in snapshot")]
-    fn post_upgrade_fails_on_global_state_invariant_check_failure() {
```

Comment:
I'm not sure this should be removed, but rather, some other bad mutation should be in the set.  This isn't a test about routing table per se.